src/crud/useGraphicFind.ts

```diff
@@ -0,0 +1,42 @@
+import { InitializedRecord } from '@orbit/records';
+import { apmGraphic } from '../components/GraphicUploader';
+import { useOrbitData } from '../hoc/useOrbitData';
+import { ArtifactCategoryD, GraphicD } from '../model';
+
+export const useGraphicFind = () => {
+  const graphics = useOrbitData('graphic') as GraphicD[];
+  const artifactCategory = useOrbitData(
+    'artifactcategory'
+  ) as ArtifactCategoryD[];
+
+  return (recId: InitializedRecord, ref?: string) => {
+    let graphicRec = graphics.find(
+      (g) =>
+        g.attributes.resourceType === recId.type &&
+        g.attributes.resourceId === parseInt(recId?.keys?.remoteId ?? '0')
+    );
+    let color = undefined;
+    if (ref) {
+      const catText = ref.split(' ')[1];
+      const catRec = artifactCategory.find(
```

Comment:
The category showing in the reference should be localized, so would need to be unlocalized to find the category from the reference

Reply:
Good catch